apps/opencs/view/render/pagedworldspacewidget.cpp

```diff
@@ -142,14 +142,15 @@ void CSVRender::PagedWorldspaceWidget::addEditModeSelectorButtons (
         "terrain-move");
 }
 
-void CSVRender::PagedWorldspaceWidget::handleMouseClick (osg::ref_ptr<TagBase> tag, const std::string& button, bool shift)
+void CSVRender::PagedWorldspaceWidget::handleMouseClick (const WorldspaceHitResult& hit, const std::string& button,
```

Comment:
I see what you mean. With what I wanted to do, this would have to call the mousePick function and may lead to the there being 2 traces. However, the result could be cached if performance is the concern.